starter-code/iteration1/src/app/app.component.html

```diff
@@ -3,3 +3,8 @@ <h1 class="title">
 </h1>
 
 <img  class="photoimage" [src]="image"/>
+
```

Comment:
Javi, this should be three photos, not one. How would you do it? (Remember, we are practicing Angular2 syntax.)